release.sh

```diff
@@ -15,6 +16,7 @@
 set -e
 
 jq --version > /dev/null || (echo "jq is required: please install it"; kill $$)
+dot-json --version > /dev/null || (echo "dot-json is required: please install it"; kill $$)
```

Comment:
This seems quite similar to the jq dependency - would it be possible to us the same for both?

Reply:
Yes, it would be possible, `dot-json` is mostly here for convenience only... With something like `jq` which only outputs to `stdout` (rather than modifying the supplied file), we would need to write to a temporary file and then move it into place (otherwise the input file is clobbered). Perhaps it's better to have just one dependency though and just make a temp file?

Reply:
I have converted to `jq` equivalent commands, thanks for the nudge!